### DISCUSSION TITLE
Bioclim extension

### DISCUSSION THREAD

**Discussion by treebreeder** [HelgeJentsch/ClimDatDownloadR#1] [General]
Hi,
the package works extremely good, I am very grateful for it.
I saw that under bioclim only the 19 bioclimatic variables are included, but not the growing degree days, etc.
Is it anyhow possible to include those additional variables and maybe even the Bioclim+ variables?

**Comment by treebreeder** [comment]
I see that it is probably because vers 2.1 was published after the package was released? But maybe worth to think about an update, since the number of variables has almost doubled under vers. 2.1...

**Comment by HelgeJentsch** [comment]
Hi @treebreeder , 

thanks for your comment and feedback. 
I'm not entirely sure if you are refering to Chelsa or WorldClim Data as both of them have a version 2.1.
It is definetly possible to implement code to add them to the downloadable indices. Sadly, I'm currently lacking the time and ressources to implement these changes properly and robustly. The sad reality of promising open source developement 😅 

I'll open an Issue about this and will try to make it work soon. 
If you have a list of the filenames, that might help me get it implemented faster. 

Cheers for the tip though!
Helge

**Reply by treebreeder** [reply]
I will provide the filenames. Unfortunately, the download service from Chelsa seems to have issues at the moment, but I will try as soon as the S3 bucket download service works again. Thanks for the support!
